### PR TITLE
at91: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
Splits up DEVICE_TITLE into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT.

### CODE REVIEW

File: target/linux/at91/image/sam9x.mk [hunk 1]
```diff
@@ -7,119 +7,139 @@ endef
...
-  DEVICE_TITLE := Calao USBA9G20
+  DEVICE_VENDOR := Calao
+  DEVICE_MODEL := USBA9G20
 endef
 TARGET_DEVICES += usb_a9g20
 
 define Device/ethernut5
   $(Device/evaluation)
-  DEVICE_TITLE := Ethernut 5
+  DEVICE_VENDOR := Ethernut
```
Comment (dangowrt) [inline target/linux/at91/image/sam9x.mk]: Should be 'egnite' according to http://www.ethernut.de/en/imprint.html

File: target/linux/at91/image/sam9x.mk [hunk 2]
```diff
@@ -7,119 +7,139 @@ endef
...
+  DEVICE_MODEL := 5
```
Comment (dangowrt) [inline target/linux/at91/image/sam9x.mk]: Should be 'Ethernut 5'

**Review by dangowrt — CHANGES_REQUESTED** [review]
Please fill correct manufacturer of the 'Ethernut 5' (see comment)

**Comment by adrianschmutzler** [comment]
Despite me not being familiar with target, this LGTM now.

**Comment by lynxis** [comment]
merged. Thank you!

**Pull request closed without merging**